homeassistant/components/sector_alarm/alarm_control_panel.py

```diff
@@ -0,0 +1,86 @@
+"""The Sector Alarm Integration."""
+import logging
+from homeassistant.components.sector_alarm import DOMAIN as SECTOR_DOMAIN
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.const import (
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_PENDING,
+)
+
+import homeassistant.components.sector_alarm as sector_alarm
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Initialition of the platform."""
+    sector_connection = hass.data.get(SECTOR_DOMAIN)
+
+    if sector_connection is None:
+        _LOGGER.error("SECTOR_DOMAIN is empty")
+        return
+
+    code = discovery_info[sector_alarm.CONF_CODE]
+    panel_id = discovery_info[sector_alarm.CONF_ALARM_ID]
+
+    async_add_entities([SectorAlarmPanel(sector_connection, panel_id, code)])
+
+
+class SectorAlarmPanel(alarm.AlarmControlPanel):
+    """Get the alarm status, and arm/disarm alarm."""
+
+    def __init__(self, sectorConnect, alarmId, code):
+        """Initialize the service."""
+        self._alarmid = alarmId
+        self._code = code
+        self._sectorconnect = sectorConnect
+        self._state = ""
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return "Sector Alarm {}".format(self._alarmid)
```

Comment:
Use the new style for string formatting -> https://developers.home-assistant.io/docs/en/development_guidelines.html#use-new-style-string-formatting